DiffusionPreprocessing/DiffPreprocPipeline_Merge.sh

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env bash
+#!/bin/bash
+#~ND~FORMAT~MARKDOWN~
+#~ND~START~
+#
+# # DiffPreprocPipeline_Merge.sh
+#
+# ## Copyright Notice
+#
+# Copyright (C) 2012-2016 The Human Connectome Project
+#
+# * Washington University in St. Louis
+# * University of Minnesota
+# * Oxford University
+#
+# ## Author(s)
+#
+# * Michiel Cottaar, FMRIB Analysis Group, Oxford University
+# * Matthew F. Glasser, Department of Anatomy and Neurobiology, Washington University in St. Louis
+# * Michael Harms, Department of Anatomy and Neurobiology, Washington University in St. Louis
+#
+# ## Product
+#
+# [Human Connectome Project][HCP] (HCP) Pipelines
+#
+# ## License
+#
+# See the [LICENSE](https://github.com/Washington-University/Pipelines/blob/master/LICENSE.md) file
+#
+# ## Description
+#
+# This script, <code>DiffPreprocPipeline_Eddy.sh</code>, implements the second
```

Comment:
This section needs to be revised to be reflective of the purpose/role of this particular script.